Controller/Comment.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @author           Serge Kishiko <sergekishiko@gmail.com>
+ * @copyright        (c) 2015-2017, Pierre-Henry Soria. All Rights Reserved.
+ * @license          Lesser General Public License <http://www.gnu.org/copyleft/lesser.html>
+ * @link             http://hizup.uk
+ */
+
+namespace TestProject\Controller;
+
+class Comment
+{
+    protected $oUtil, $oModel;
+
+    public function __construct()
+    {
+        // Enable PHP Session
+        if (empty($_SESSION))
+            @session_start();
+
+        $this->oUtil = new \TestProject\Engine\Util;
+
+        /** Get the Model class in all the controller class **/
+        $this->oUtil->getModel('Comment');
+        $this->oModel = new \TestProject\Model\Comment;
```

Comment:
And then, maybe rename `$oModel` to `oCommentModel` to avoid confusion with its parent Blog class